modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/WorkspaceExtension.java

```diff
@@ -272,6 +312,100 @@ public void setTargetPlatformVersion(Object targetPlatformVersion) {
 		_targetPlatformVersion = targetPlatformVersion;
 	}
 
+	public void setWorkspaceProductKey(Object workspaceProductKey) {
+		_product = workspaceProductKey;
```

Comment:
The `product` property is named differently in a few places.  Can we rename it to `productKey` everywhere?

```
public void setProductKey(Object productKey) {
	_productKey = productKey;
}
```